32blit-sdl/System.cpp

```diff
@@ -269,6 +272,16 @@ void System::loop()
 	blit::joystick.y = shadow_joystick[1];
 	SDL_UnlockMutex(m_input);
 	blit::tick(::now());
+
+    if(blit::vibration > 0) {
```

Comment:
Doesn't this mean you can't turn vibration off? Though the duration would make it auto-disable anyway... This block could be an `update_rumble` method on `Input` or something similar.

Reply:
Nopes, no need to. The code only rumbles for 20ms max anyway. So it only rumbles until the next cycle. And then either is restarted for another 20ms or silenced automatically.